test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -733,6 +733,254 @@ def check_sizing_values(model, building_type, template, climate_zone)
+      building_type, template, climate_zone, user_data_dir, mods = prototype
+
+      # to simplify testing procedures, for all test cases below, the following are true
+      #   - zone area is larger than 500 sqft
+      #   - air loop has economizer
+
+      tc_ids = nil
+      mods.each do |mod|
+        if mod[0] == 'mark_test_case_no'
+          tc_ids = mod[1]
+        end
+      end
+      if tc_ids.nil?
+        assert(false, 'mark_test_case_no mod not set, cannot proceed with DCV test check_dcv')
+      end
+
+      tc_ids.each do |tc_id|
+        case tc_id
+        when 1
+          # test case 1:
+          #   - DCV should be in the user model (zone ppl density > 25 ppl/ksqft, no user exception)
+          #   - DCV should be in the baseline (air loop oa flow > 3000 cfm && zone ppl density > 100 ppl/ksqft)
+          #   - DCV is implemented in user model
+          # expected result: baseline implements DCV
+          # test case setting:
+          #   - Cafeteria
+          #   - user model air loop oa flow 3153 cfm
+          #   - no user data needed
+          #   - zone ppl density 101 [through ppl density modifier]
+          #   - DCV implemented in the user model
+          #     - at the airloop level [through modifier]
+          #     - at the zone level, zone oa spec per person 0.003539605824
+          zone = model_baseline.getThermalZoneByName('Cafeteria_ZN_1_FLR_1 ZN').get
+          airloop = zone.airLoopHVAC.get
+          assert(dcv_is_on(zone, airloop))
+
+        when 2
+          # test case 2:
+          #   - DCV should not be in the user model (zone ppl density > 25 ppl/ksqft, but has ZONE user exception)
+          #   - DCV should be in the baseline (air loop oa flow > 3000 cfm && zone ppl density > 100 ppl/ksqft)
+          #   - DCV is implemented in user model
+          # expected result: baseline implements DCV but prompts warning (user model has DCV but meet exception)
+          # test case setting:
+          #   - Cafeteria
+          #   - user model air loop oa flow 3153 cfm
+          #   - zone ppl density 101 [through ppl density modifier]
+          #   - user data specifies ZONE DCV exception is true
+          #   - DCV implemented in the user model
+          #     - at the airloop level [through modifier]
+          #     - at the zone level, zone oa spec per person 0.003539605824
+          zone = model_baseline.getThermalZoneByName('Cafeteria_ZN_1_FLR_1 ZN').get
+          airloop = zone.airLoopHVAC.get
+          # check warning
+          assert(dcv_is_on(zone, airloop))
+
+        when 3
+          # test case 3:
+          #   - DCV should be in the user model (zone ppl density > 25 ppl/ksqft, no user exception)
+          #   - DCV should be in the baseline model (air loop oa flow > 3000 cfm && zone ppl density > 100 ppl/ksqft)
+          #   - DCV is NOT implemented in user model
+          # expected result: error and terminate
```

Comment:
Only terminate if zone OA requirement is not specified as flow per person. Otherwise, apply DCV to baselne.